src/GraphQLParser/LexerContext.cs

```diff
@@ -48,38 +46,26 @@ public Token GetToken()
             if (code == '"')
```

Comment:
A few lines above, you can change `char.IsLetter(code)` to `('a' <= code && code <= 'z') || ('A' <= code && code <= 'Z')` for faster performance.  Similar with `IsNumber`.  No need to check for unicode variations of the alphabet if the spec doesn't allow for it.

Reply:
done